src/commands/register.ts

```diff
@@ -36,6 +36,10 @@ export default class ComponentRegister extends Command {
       description: 'Tag to give to the new component',
       default: 'latest',
     }),
+    platform: Flags.string({
+      description: 'Platform(s) you are running on',
```

Comment:
Showcase how to do multiple platforms at the same time.

Reply:
The property `multiple: true,` would probably be useful here

Reply:
Good idea. I added `multiple` to the flag.